src/hardware/openbench-logic-sniffer/protocol.c

```diff
@@ -596,7 +596,7 @@ SR_PRIV int ols_prepare_acquisition(const struct sr_dev_inst *sdi) {
 		RETURN_ON_ERROR(ols_send_reset(serial));
 
 		delaycount = readcount * (1 - devc->capture_ratio / 100.0);
-		devc->trigger_at_smpl = (readcount - delaycount) * 4 - basic_trigger_desc.num_stages;
+		devc->trigger_at_smpl = (readcount - delaycount) * 4 - 1;
```

Comment:
So, to double check: you had all basic triggers activated and it didn't make a difference. It would make sense because the delay is the one before the first trigger, or?

Reply:
Yes, I progressively added more basic trigger stages and noticed that the old code shifted the trigger point, whereas this code keeps it constant.

Reply:
FYI: This commit has now hash 45d4e9e8321e51abdee2c29955b2ec9285ae0918, if you want to continue from there.